manual/src/cmds/runtime-tracing.etex

```diff
@@ -205,6 +234,83 @@ instead, call the program as below to produce the same result:
 OCAML_RUNTIME_EVENTS_START=1 ./example
 \end{verbatim}
 
+\subsubsection{s-runtime-tracing-custom-events}{Tracing custom events}
+
+The following program make use of the \texttt{Runtime_events.User} and
+\texttt{Runtime_events.Type} modules to declare a custom event providing
+\texttt{span} and \texttt{int} values. The \texttt{tag} extensible variant is
```

Comment:
Is this one custom event providing two values, or two custom events, one with a span and one with an int value?

Reply:
Two events indeed.